ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -49,6 +49,14 @@ func ResourceIBMContainerVpcWorkerPool() *schema.Resource {
 					"cluster"),
 			},
 
+			"default_worker_pool": {
+				Type:        schema.TypeBool,
+				Required:    false,
+				ForceNew:    false,
+				Optional:    true,
+				Description: "If true, instead of creating one, the create function will read the the clsuter's \"default\" workerpool ",
+			},
```

Comment:
maybe it can be general. what do you think?
```suggestion
			"import_worker_pool": {
				Type:        schema.TypeBool,
				Required:    false,
				ForceNew:    false,
				Optional:    true,
				Description: "If true, instead of creating one, the create function will try to import an existing workerpool",
			},
```